src/components/Infobar/Infobar.scss

```diff
@@ -14,11 +15,45 @@
   z-index: $infobar-z-index;
 }
 
-.info-bar__return-to-presented-note-button > svg {
-  background-color: $menu-icon-background-color--dark;
+.info-bar__return-to-shared-note-button {
+  position: relative;
+  border: 0;
+  background-color: $color-white;
+  box-shadow: $box-shadow--light;
+  border-radius: $rounded--full;
+  height: 42px;
+  width: 42px;
+  color: white;
+  cursor: pointer;
+  display: grid;
+  place-content: center;
+  outline: none;
+  transition: background-color 150ms linear;
+  color: $color-black;
+
+  &:hover {
+    background-color: color.scale($color-white, $lightness: -10%);
```

Comment:
Wouldn't it be better to define this scaling once and then apply it to both info-bar buttons (i.e. return to shared note and add one minute to the timer) for consistency?
It may be used somewhere else, too.